lib/authoritex.ex

```diff
@@ -47,7 +47,14 @@ defmodule Authoritex do
      %{id: "info:lc/authorities/names/no2009021335", label: "Melo, Cimara"}
    ]}
   """
-  def search(authority_code, query, max_results \\ 30) do
+  def search(authority_code, query) do
+    case(find_authority(authority_code)) do
+      nil -> {:error, "Unknown authority: #{authority_code}"}
+      authority -> authority.search(query)
+    end
+  end
+
+  def search(authority_code, query, max_results) do
```

Comment:
I think we need to duplicate [this assertion](https://github.com/nulib/authoritex/blob/fast/test/authoritex_test.exs#L78-L79) to test both `search/2` and `search/3` now that they're no longer the same function with a default.